config/routes.rb

```diff
@@ -104,6 +104,8 @@
   root :to => 'assets#latest'
 
   resources :users
+
+  get '/invite/:mass_invite_token', to: 'mass_invites_users#new', as: 'mass_invite'
```

Comment:
So what are you going to use for single-use invites 😄

Reply:
Heh yeah, I know... I wanted to live with it for a day because this is the ideal URL...

Reply:
Was thinking of using a route constraint to check token length or for a token prefix